src/app/components/normal-component/count-down/count-down.component.spec.ts

```diff
@@ -93,34 +90,34 @@ describe('CountDownComponent', () => {
         component.setDuration(3000);
         component.start();
         tick(1000);
-        fixture.detectChanges();
-        let timeText: string = fixture.debugElement.query(By.css('#remainingTime')).nativeElement.innerHTML;
+        testUtils.detectChanges();
+        let timeText: string = testUtils.findElement('#remainingTime').nativeElement.innerHTML;
         expect(timeText).toBe('0:02');
         tick(1000);
-        fixture.detectChanges();
-        timeText = fixture.debugElement.query(By.css('#remainingTime')).nativeElement.innerHTML;
+        testUtils.detectChanges();
+        timeText = testUtils.findElement('#remainingTime').nativeElement.innerHTML;
         expect(timeText).toBe('0:01');
         component.stop();
     }));
     it('should update written time correctly (closest rounding) even when playing in less than refreshing time', fakeAsync(() => {
         spyOn(component.outOfTimeAction, 'emit').and.callThrough();
         component.setDuration(599501); // 9 minutes 59 sec 501 ms
```

Comment:
pourquoi pas `.setDuration((9 * 60 + 59) * 1000 + 501)` ? ainsi pas de risque que le commentaire soit out of sync